### ISSUE TITLE
:yank markdown (and asciidoc and html)

### ISSUE THREAD

**Issue by nancym** [qutebrowser/qutebrowser#4647]
I love the [v.1.6.0](https://github.com/qutebrowser/qutebrowser/releases/tag/v1.6.0) `:yank markdown` feature - thank you for that. For those of us who write in asciidoc and html, it would be great to also be able to yank in those formats, i.e., have this:

* :yank markdown
  * `[DuckDuckGo — Privacy, simplified.](https://start.duckduckgo.com/)`
* :yank asciidoc
  * `https://start.duckduckgo.com/[DuckDuckGo — Privacy, simplified.]`
* :yank html
  * `<a href="https://start.duckduckgo.com/">DuckDuckGo — Privacy, simplified.</a>`

Thank you for considering this!

**Comment by jgkamat** [comment]
FWIW, I don't think :yank markdown should be in core. This feature makes more
sense as a plugin to me.

Actually, now that I think about it, you should already be able to do all
these things with userscripts.

**Comment by ninewise** [comment]
Depending on "xclip", the userscript below does the asciidoc version.

```sh
#!/bin/sh

# in hints mode, the text of the selected hint
# in command mode, the selected text if any, the page title otherwise
printf "%s[%s]" "$QUTE_URL" "${QUTE_SELECTED_TEXT:-$QUTE_TITLE}" | xclip
```

**Comment by nancym** [comment]
Thanks for the userscript @ninewise. Does anyone have any tips for userscripts on Windows 10? I've been searching the web for examples and haven't found any 😞

**Comment by jgkamat** [comment]
n writes:

> Thanks for the userscript @ninewise. Does anyone have any tips for userscripts on Windows 10? I've been searching the web for examples and haven't found any 😞

From the documentation

> Note: On Windows, only userscripts with com, bat, or exe extensions will be launched.

Although, I haven't been able to get userscripts working on windows personally.

**Comment by user202729** [comment]
Actually, all of these features are a subset of #1862 (ignoring escaping quotes etc.)

`:yank markdown` -> `:yank [{title}]({url})`
`:yank asciidoc` -> `:yank {url}[{title}]`
`:yank html` -> `:yank <a href="{url}">{title}</a>`

**Comment by The-Compiler** [comment]
Fixed via #4651.
